OrbitCore/ElfFileTests.cpp

```diff
@@ -15,17 +15,17 @@ TEST(ElfFile, LoadFunctions) {
 
   Pdb pdb;
   ASSERT_TRUE(elf_file->LoadFunctions(&pdb));
-  std::vector<Function>& functions = pdb.GetFunctions();
+  const std::vector<std::shared_ptr<Function>>& functions = pdb.GetFunctions();
   EXPECT_EQ(functions.size(), 10);
-  const Function* function = &functions[0];
+  const Function* function = functions[0].get();
```

Comment:
maybe const auto& function = functions[0], you can use shared_ptr below...

Reply:
function is reused below, and I can't declare as non-const ref...  will leave as is.